dashboard/test/factories/pd_factories.rb

```diff
@@ -616,7 +616,7 @@
   factory :pd_facilitator1819_application, class: 'Pd::Application::Facilitator1819Application' do
     association :user, factory: :teacher, strategy: :create
     transient do
-      form_data_hash {build :pd_facilitator1819_application_hash}
+      form_data_hash {build :pd_facilitator1819_application_hash, program: Pd::Application::Facilitator1819Application::PROGRAMS[course.to_sym]}
```

Comment:
Doesn't `course`  need a default factory value to be referenced here? I suspect this will break any existing tests that don't specify a course.

Reply:
Adding one - lets see if this breaks anything